ui/apps/platform/src/Containers/ComplianceEnhanced/Schedules/components/ScanConfigClustersTable.tsx

```diff
@@ -92,51 +96,42 @@ function ScanConfigClustersTable({ clusterScanStatuses }: ScanConfigClustersTabl
     const clustersWindow = sortedClusters.slice(startNumber, endNumber);
 
     return (
-        <Card>
-            <CardHeader
-                actions={{
-                    actions: (
-                        <>
-                            <Pagination
-                                itemCount={clusterScanStatuses.length}
-                                page={page}
-                                onSetPage={onSetPage}
-                                perPage={perPage}
-                                onPerPageSelect={onPerPageSelect}
-                            />
-                        </>
-                    ),
-                    hasNoOffset: false,
-                    className: undefined,
-                }}
-            >
-                <CardTitle component="h2">Clusters</CardTitle>
-            </CardHeader>
-            <CardBody>
-                <Table borders={false}>
-                    <Thead noWrap>
-                        <Tr>
-                            <Th sort={getSortParams(0)}>Cluster</Th>
-                            <Th sort={getSortParams(1)} width={20}>
-                                Operator status
-                            </Th>
-                        </Tr>
-                    </Thead>
-                    <Tbody>
-                        {clustersWindow.map((cluster) => {
-                            return (
-                                <Tr key={cluster.clusterId}>
-                                    <Td dataLabel="Cluster">{cluster.clusterName}</Td>
-                                    <Td dataLabel="Operator status">
-                                        <ComplianceClusterStatus errors={cluster.errors} />
-                                    </Td>
-                                </Tr>
-                            );
-                        })}
-                    </Tbody>
-                </Table>
-            </CardBody>
-        </Card>
+        <Flex>
+            <Flex grow={{ default: 'grow' }}>
+                <Title headingLevel={headingLevel}>Clusters</Title>
```

Comment:
Opinionated comment: I like what you did with the total count badges and although I understand why you didn't include it here I still think it would be nice to have. Either way works for me though!

Reply:
Yeah, absence of count badge looks inconsistent. We can discuss and improve in planned follow up.